prepare.py

```diff
@@ -121,3 +120,10 @@ def prepare_netgen_plugin():
     prepare_geom()
     prepare_smesh()
     prepare_netgen_plugin()
+
+    # throw -> noexcept
+    os.system("grep -rl 'throw(SALOME_Exception)' ./src | xargs sed -i 's/throw(SALOME_Exception)/noexcept(false)/g'")
+    os.system("grep -rl 'throw( SALOME_Exception )' ./src | xargs sed -i 's/throw( SALOME_Exception )/noexcept(false)/g'")
+    os.system("grep -rl 'throw (SALOME_Exception)' ./src | xargs sed -i 's/throw (SALOME_Exception)/noexcept(false)/g'")
+    os.system("grep -rl 'throw ( SALOME_Exception )' ./src | xargs sed -i 's/throw ( SALOME_Exception )/noexcept(false)/g'")
+    os.system("grep -rl 'throw (std::bad_alloc)' ./src | xargs sed -i 's/throw (std::bad_alloc)/noexcept(false)/g'")
```

Comment:
Can you put in it's own function? Something like `prepare_noexcept`? I'm just trying to keep the `__main__` part of it to just calling functions. When I was debugging things locally I found myself commenting out specific "prepare" functions, so it might be nice to have this in the same style.

Reply:
I think from and NETGENPlugin have also some cases which get replaced.

Reply:
@trelau the find replace should be done on all the directories. any ideas?

Reply:
just move this chunk of code into its own function `prepare_noexcept` called after `prepare_netgen_plugin`?